consensus/scp/src/node.rs

```diff
@@ -152,14 +148,14 @@ pub trait ScpNode<V: Value>: Send {
     fn get_slot_metrics(&mut self, slot_index: SlotIndex) -> Option<SlotMetrics>;
 
     /// Get the slot internal state (for debug purposes).
-    fn get_slot_state(&mut self, slot_index: SlotIndex) -> Option<SlotState<V>>;
+    fn get_slot_state(&mut self, slot_index: SlotIndex) -> Option<String>;
```

Comment:
Previously, this returned a SlotState, which was serialized to JSON and logged. That created an implicit dependency between Node and Slot. Here, the conversion to JSON has been pushed down so that Node is not aware of SlotSate.

Reply:
maybe `get_slot_debug_snapshot()`?